icedb/icedb.py

```diff
@@ -340,34 +340,46 @@ def tombstone_cleanup(self, min_age_ms: int) -> tuple[list[str], list[str], list
         now = round(time() * 1000)
 
         log_files_to_delete: dict[str, bool] = {}
+        log_files_to_keep: dict[str, LogTombstone] = {}
         data_files_to_delete: dict[str, bool] = {}
         data_files_to_keep: dict[str, FileMarker] = {}
         schema = Schema()
 
-        current_log_files = logio.get_current_log_files(self.log_s3c)
+        cur_schema, cur_files, cur_tombstones, all_log_files = logio.read_at_max_time(self.log_s3c, round(time() * 1000))
```

Comment:
```suggestion
        cur_schema, cur_files, cur_tombstones, all_log_files = logio.read_at_max_time(self.log_s3c, round(time() * 1000))
```
This should use `now`, not another (later) time